docs/host-embedding/host-embedding.html

```
@@ -0,0 +1,847 @@
+<title>Cosmos Host Embedding</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)